dist/bash_completion/rkt.bash

```diff
@@ -757,6 +757,28 @@ _rkt_status()
     must_have_one_noun=()
 }
 
+_rkt_stop()
+{
+    last_command="rkt_stop"
+    commands=()
+
+    flags=()
+    two_word_flags=()
+    flags_with_completion=()
+    flags_completion=()
+
+    flags+=("--debug")
+    flags+=("--dir=")
+    flags+=("--insecure-options=")
+    flags+=("--insecure-skip-verify")
+    flags+=("--local-config=")
+    flags+=("--system-config=")
+    flags+=("--trust-keys-from-https")
```

Comment:
This needs to be updated with `--force`.

(It would be easier with https://github.com/coreos/rkt/issues/1974)